src/Sulu/Component/HttpCache/EventSubscriber/ContentMapperSubscriber.php

```diff
@@ -0,0 +1,86 @@
+<?php
+/*
+ * This file is part of the Sulu CMS.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\HttpCache\EventSubscriber;
+
+use Sulu\Component\Content\Event\ContentNodeEvent;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Sulu\Component\Content\ContentEvents;
+use Sulu\Component\HttpCache\StructureCacheHandlerInterface;
+use Sulu\Component\Content\Event\ContentNodeDeleteEvent;
+use Sulu\Component\HttpCache\HandlerInterface;
+use Sulu\Component\HttpCache\HandlerInvalidateStructureInterface;
+
+/**
+ * Listen to the content mapper and invalidate structures
+ */
+class ContentMapperSubscriber implements EventSubscriberInterface
+{
+    /**
+     * @var Sulu\Component\HttpCache\StructureCacheHandlerInterface
+     */
+    private $handler;
+
+    /**
+     * @var StructureInterface[]
+     */
+    private $structureInvalidationStack = array();
+
+    /**
+     * {@inheritDoc}
+     */
+    public static function getSubscribedEvents()
+    {
+        return array(
+            ContentEvents::NODE_POST_SAVE => 'onContentNodeSave',
```

Comment:
I guess this is not part of this PR, but shouldn't this event be called `onContentNodePostSave`?